server/server.go

```diff
@@ -56,7 +56,7 @@ func (keys *defaultKeystore) Add(op *gokeyless.Operation, priv crypto.Signer) er
 	keys.Lock()
 	defer keys.Unlock()
 
-	if digest, ok := gokeyless.GetDigest(priv.Public()); ok {
+	if digest, err := gokeyless.GetDigest(priv.Public()); err == nil {
```

Comment:
No, if GetDigest will only succeed for RSA keys. We should just discard the error it throws for ECDSA keys (and not have a digest registered for that key).